llvm/test/CodeGen/EraVM/overflow.ll

```diff
@@ -0,0 +1,117 @@
+; RUN: llc < %s | FileCheck %s
```

Comment:
Please add tests with all variants of select instructions, since here we are only checking for few cases. Also, please add pre-commit test without changes, and after that this support to see the actual improvements (E.g. pre-commit https://github.com/matter-labs/era-compiler-llvm/commit/0604f9f7726d393aa602f3c40b931a0c34885823, actual commit https://github.com/matter-labs/era-compiler-llvm/commit/7311da6251e108301d1d9b545e1386f03313f838). Don't manually create checks, please use `update_llc_test_checks.py` to do so:
`llvm/llvm/utils/update_llc_test_checks.py --llc-binary target-llvm/build-final/bin/llc llvm/llvm/test/CodeGen/EraVM/overflow.ll`

Reply:
I see now. Will address this part tomorrow.

Reply:
Hi @vladimirradosavljevic , I create a separate PR #560, would you please take a look?

Reply:
The pre-commit tests are added now.